Resources/English.lproj/InfoPlist.strings

```diff
@@ -1,3 +1,3 @@
 ﻿/* Localized versions of Info.plist keys */
 
-NSHumanReadableCopyright = "© Pieter de Bie, 2010";
\ No newline at end of file
+NSHumanReadableCopyright = "© GitX(N) developers, 2017";
```

Comment:
Per https://tldrlegal.com/license/gnu-general-public-license-v2 we can't do this. Can someone correct me?

Reply:
My interpretation is (maybe I'm wrong) : we can do this 
<img width="588" alt="image" src="https://user-images.githubusercontent.com/3314607/153193604-5b316da6-1b61-44f7-a0d2-dca571953f2c.png">

our changes are **_available_** via git history